.github/workflows/daily-e2e-tests-ibmcloud.yaml

```diff
@@ -89,6 +90,9 @@ jobs:
             s390x-secure-execution)
               arch_string="s390x-se"
               ;;
+            amd64-csi-wrapper)
+              arch_string="amd64"
```

Comment:
for csi-wrapper, case, we do NOT need download and push podvm image again, as we use the same image from `amd64` type.